popup/popup.js

```diff
@@ -228,19 +238,32 @@ function showStyles(styles) {
     return;
   }
 
-  const enabledFirst = prefs.get('popup.enabledFirst');
-  styles.sort((a, b) => (
-    enabledFirst && a.enabled !== b.enabled
-      ? !(a.enabled < b.enabled) ? -1 : 1
-      : a.name.localeCompare(b.name)
-  ));
-
   const container = document.createDocumentFragment();
-  styles.forEach(style => createStyleElement({style, container}));
+  sortStyles({
+    styles,
+    getEnabled: entry => entry.enabled,
+    getName: entry => entry.name
+  })
+  .forEach(style => createStyleElement({style, container}));
+
   installed.appendChild(container);
   window.dispatchEvent(new Event('showStyles:done'));
 }
 
+function sortStylesInPlace() {
+  if (prefs.get('popup.autoResort')) {
```

Comment:
Also `sortStylesInPlace` can be simplified to:
```js
if (pref.get('popup.enabledFirst') && pref.get('popup.autoResort')) {
	$$('.entry', installed).sort(compareEnabled)
		.forEach(e => installed.appendChild(e));
}
```
BTW, is it safe to replace `$$('.entry', installed)` with `[...installed.children]`?

Reply:
Using `[...installed.children]` would include the hotkey aside (the first element)

```html
<aside id="hotkey-info" title="Click to see available hotkeys"></aside>
```